src/vm/arm/stubs.cpp

```diff
@@ -544,7 +544,12 @@ void LazyMachState::unwindLazyState(LazyMachState* baseState,
             DacError(hr);
         }
 #else // DACCESS_COMPILE
-        PAL_VirtualUnwind(&ctx, &nonVolRegPtrs);
+        BOOL success = PAL_VirtualUnwind(&ctx, &nonVolRegPtrs);
+        if ( !success )
```

Comment:
Nit - the typical style in CoreCLR is `if (!success)` - not spaces inside parenthesis.